index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
```

Comment:
A 'viewport' meta element was not specified. Please use the correct format at stated on the [Semantic Html](https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md)